python/inputmodule/gui/ledmatrix.py

```diff
@@ -72,7 +80,9 @@ def random_eq(dev):
 def clock(dev):
     """Render the current time and display.
     Loops forever, updating every second"""
-    while True:
+    animate(dev, False)
+    set_status('clock')
+    while True and get_status() == 'clock':
```

Comment:
```suggestion
    while get_status() == 'clock':
```
Even more nitpicking :sweat_smile: